src/GRA.Web/Views/Events/_DetailPartial.cshtml

```diff
@@ -1,8 +1,10 @@
 ﻿@model GRA.Controllers.ViewModel.Events.EventsDetailViewModel
 
+@inject Microsoft.Extensions.Localization.IStringLocalizer<GRA.Resources.Shared> SharedLocalizer
```

Comment:
SharedLocalizer doesn't need to be injected here as it's coming in through the _ViewImports.

Reply:
Fixed in commit a644d762f46ed05c11f0f643e52b135003a8f354